cg/store/models.py

```diff
@@ -975,6 +976,24 @@ class RunDevice(Base):
     }
 
 
+class IlluminaFlowCell(RunDevice):
+    """Model for storing Illumina flow cells."""
+
+    __tablename__ = "illumina_flow_cell"
+
+    id: Mapped[int] = mapped_column(ForeignKey("run_device.id"), primary_key=True)
+    model: Mapped[IlluminaFlowCellModels | None] = mapped_column(
+        types.Enum(
+            IlluminaFlowCellModels,
+            values_callable=lambda: [
```

Comment:
Im getting `E   TypeError: IlluminaFlowCell.<lambda>() takes 0 positional arguments but 1 was given` with this alternative, will try the other one

Reply:
switched to the other approach